### ISSUE TITLE
Thumbnail for png images do not show

### ISSUE THREAD

**Issue by cheeka13** [nextcloud/android#14265]
### ⚠️ Before posting ⚠️

- [x] This is a **bug**, not a question or an enhancement.
- [x] I've [searched for similar issues](https://github.com/nextcloud/android/issues) and didn't find a duplicate.
- [x] I've written a clear and descriptive title for this issue, not just "Bug" or "Crash".
- [x] I agree to follow Nextcloud's [Code of Conduct](https://nextcloud.com/contribute/code-of-conduct/).

### Steps to reproduce

1. Upload a .png file
2. Click the file to load
3. Back to All files or media tab

### Expected behaviour

Thumbnail of the image shown

### Actual behaviour

Default thumbnail

### Android version

13

### Device brand and model

any android device

### Stock or custom OS?

Stock

### Nextcloud android app version

3.30.0 Alpha1

### Nextcloud server version

Newest

### Using a reverse proxy?

I don't know

### Android logs

![Image](https://github.com/user-attachments/assets/32d8b936-3803-49b1-a71b-eaae595bf0f7)

### Server error logs

```bash

```

**Comment by alperozturk96** [comment]
https://github.com/nextcloud/android/pull/14025

**Comment by cheeka13** [comment]
> [#14025](https://github.com/nextcloud/android/pull/14025)

Why you closed? The bug is still there and I cannot see thumbnails of png images

**Comment by cheeka13** [comment]
> [#14025](https://github.com/nextcloud/android/pull/14025)

You understand me? Open the issue again

**Comment by alperozturk96** [comment]
> > [#14025](https://github.com/nextcloud/android/pull/14025)
> 
> You understand me? Open the issue again

The issue has been resolved with the [provided fix](https://github.com/nextcloud/android/pull/14025), which will be included in the upcoming release.

**Comment by alperozturk96** [comment]
A new pull request has been created to address this issue. Once it is merged, the issue will be automatically closed, and the fix will be included in the next release.

**Comment by cheeka13** [comment]
> [A new pull request ](https://github.com/nextcloud/android/pull/14270)has been created to address this issue. Once it is merged, this issue will be automatically closed, and the fix will be included in the next release.
> 
> You can stay updated on upcoming releases by visiting our[ release page](https://github.com/nextcloud/android/releases). The latest version is 3.30.6.
> 
> If you would like, you can [test the fix ](https://github.com/nextcloud/android/pull/14270#issuecomment-2565617648)and share your feedback on whether the issue has been resolved.

Thank you so much, it works
